src/Microsoft.AspNetCore.Mvc.Testing/build/netstandard2.0/Microsoft.AspNetCore.Mvc.Testing.targets

```diff
@@ -15,17 +15,17 @@
     <PreserveCompilationContext>true</PreserveCompilationContext>
   </PropertyGroup>
 
-  <PropertyGroup Condition=" '$(TargetFramework)' != 'netcoreapp2.0' ">
-    <!-- Work around https://github.com/dotnet/sdk/issues/926. Align with bitness of the web site projects. -->
-    <PlatformTarget>x86</PlatformTarget>
-
-    <!--
-      Work around https://github.com/Microsoft/vstest/issues/428 aka https://github.com/aspnet/Mvc/issues/5873.
-      Create the appropriate binding redirects.
-    -->
-    <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
-    <GenerateBindingRedirectsOutputType>true</GenerateBindingRedirectsOutputType>
-  </PropertyGroup>
+  <Target 
+      Name="AddContentRootForProjectReferences"
+      BeforeTargets="BeforeCompile"
+      DependsOnTargets="PrepareForBuild;GetReferenceAssemblyPaths">
+    <ItemGroup>
+      <AssemblyAttribute Include="System.Reflection.AssemblyMetadataAttribute">
+        <_Parameter1>Microsoft.AspNetCore.Testing.ContentRoot[$([System.IO.Path]::GetFileName(%(_ResolvedProjectReferencePaths.FullPath)))]</_Parameter1>
```

Comment:
You'll have to call `ResolveProjectReferences` and read the `TargetOutputs` to get the project reference paths. You should not use `_ResolvedProjectReferencePaths` since it's non-public.

Reply:
Can you tell me what to change? I’m using what was there. Wouldn’t calling It again affect the perf of the build?

Reply:
https://github.com/Microsoft/msbuild/blob/master/documentation/ProjectReference-Protocol.md has notes on the target. You're right -`ResolveProjectReferences` does build the project by default, but we could turn it off by setting `BuildProjectReferences = false`.

Reply:
I'm not sure I understand what you mean. That document doesn't clarify anything to me. What MSBuild do I write to do what you are asking me to. I can't think of a way to do this the way you imply.